capstone/src/main/webapp/index.html

```diff
@@ -1,30 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta name="google-signin-client_id" content="264953829929-ffv5q12afu4a7bm3vd7pnlfg8d96tlom.apps.googleusercontent.com" charset="UTF-8">
     <title>ClubHub</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-    <script src="explore-loader.js"></script>
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <script src="profile-loader.js"></script>
   </head>
-  <body>
-    <div id="top-navigation"></div>
-    <h1 id="explore-heading">Explore Clubs</h1>
-    <div id="content">
-      <div id="club-listings" class="full-screen-panel">
-        <template id="club-listing">
-          <div class="club-logo-container">
-            <img src="" id="club-logo" class="club-logo" alt="Club logo">
-          </div>
-          <div class="club-info-container">
-            <h2><a href="" id="club-name"></a></h2>
-            <p id="description"></p>
-            <p id="members"></p>
-          </div> 
-          <form id="club-join-container" name="join" action="/student-data" method="POST"></form>
-        </template>
-      </div>
-      <button id="register-club-button" onclick="location.href='club-registration.html';">Register a new club!</button>
-    </div>
+  <body id="login">
```

Comment:
Under this migration of the explore page, will already logged in users still have to go to this page first, or will index.html redirect to explore.html?

Reply:
So when someone first goes to the site, they have an initial login page and if you're already logged in, it will say you're "signed in" and take you to the explore page. So everyone will see the front page, whether logged in or not